bcda/logging/middleware.go

```diff
@@ -15,6 +16,12 @@ import (
 func NewStructuredLogger() func(next http.Handler) http.Handler {
 	logger := logrus.New()
 	logger.Formatter = &logrus.JSONFormatter{}
+	file, err := os.OpenFile("/var/log/bcda-request.log", os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
Please make the location of the file something we can pass by setting an env var (e.g., `BCDA_REQUEST_LOG`). If the env var is not set, fallback to `logger` default.